org.coolreader_ng.crqt-ng.yml

```diff
@@ -0,0 +1,93 @@
+app-id: org.coolreader_ng.crqt-ng
```

Comment:
`coolreader-ng.org` doesn't respond. Is this domain under the control of the upstream developers for their project?

Reply:
Given that the appdata from upstream list `https://gitlab.com/coolreader-ng/crqt-ng` as the homepage, a proper app-id would be `io.gitlab.coolreader_ng.crqt-ng`

Reply:
> Is this domain under the control of the upstream developers for their project?

Yes, but site is down.

> a proper app-id would be io.gitlab.coolreader_ng.crqt-ng

Ok, if I rename manifest file, should I close this PR & open new one?

Reply:
No, and if you control said domain, you don't need to do anything.

Reply:
as mentioned if you own/control the domain there is no change required.

Just there was no other mention of it anywhere and it was down so I was asking.

Reply:
But I still decided to rename the application domain.